stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/ChatTheme.kt

```diff
@@ -76,6 +77,11 @@ private val LocalMessageAlignmentProvider = compositionLocalOf<MessageAlignmentP
     error("No MessageAlignmentProvider provided! Make sure to wrap all usages of Stream components in a ChatTheme.")
 }
 
+// TODO - use the same class from the UI components once implemented
```

Comment:
@MarinTolic Once your PR is merged and aligned we can reuse the class to represent this. (if it hasn't been merged already)